node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -387,11 +389,20 @@ object Configuration {
       case Some(options.deployDemo) => DeployDemo
       case Some(options.propose)    => Propose
       case Some(options.showBlock)  => ShowBlock(options.showBlock.hash())
-      case Some(options.showBlocks) => ShowBlocks
-      case Some(options.run)        => Run
-      case Some(options.dataAtName) => DataAtName(options.dataAtName.name())
-      case Some(options.contAtName) => ContAtName(options.contAtName.name())
-      case _                        => Help
+      case Some(options.showMainChain) =>
+        import options.showMainChain._
+        ShowMainChain(depth.getOrElse(1L))
+      case Some(options.run) => Run
+      case Some(options.dataAtName) =>
+        import options.dataAtName._
+        DataAtName(
+          depth.getOrElse(default = Long.MaxValue),
+          options.dataAtName.name()
```

Comment:
`options.dataAtName.name()`, but you already have `import options.dataAtName._`.

Reply:
Done